### DISCUSSION TITLE
Http-Based Plugin

### DISCUSSION THREAD

**Discussion by st0le** [Flow-Launcher/Flow.Launcher#1307] [Ideas]
Hello @jjw24 , would you be willing to accept a PR for HTTP based plugin? I'm sort of working on it in my free time. (See my st0le/http-plugin branch in my fork)

Instead of running a executable, it would send the json payload to a REST API. I think it would help lower the entry barrier for plugin development. Anyone can stick a boilerplate plugin and just change the REST API to give FL new capabilities.

To clarify, the plugin follows the same protocol.

**Comment by jjw24** [comment]
Hi @st0le how's it going? I couldn't see it in your repos, would you mind linking me please.

**Reply by st0le** [reply]
Made a draft PR #1311
